jobs/openshift/image-registry/deploy.groovy

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env groovy
+
+library "github.com/stevekuznetsov/release-library@skuznets/image-builds"
+
+deployClusterPipeline()
```

Comment:
@csrwng realizing that we just need one global deploy pipe for projects that are not the installer?